checkpoint/genesis.go

```diff
@@ -45,6 +46,10 @@ func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) {
 		}
 	}
 
+	if err := keeper.MigrateCheckpointKeyV2(ctx); err != nil {
```

Comment:
TODOs:
is this a good place to run the migration?
is InitGenesis guaranteed to be run on process restart?